src/scripts/hooks/before.ts

```diff
@@ -18,4 +19,5 @@ export default async function (capabilities: object[], specs: string[], browser:
   //Add authenticators
   await authenticatorHandler.attachAuthHandling();
 
+  await Ui5ExtensionMocker.mockRequests();
```

Comment:
we definitely need a config flag to disable mocking in case someone needs these components.

I suggest adding a new config param like:
```ts
qmate: {
  enableUi5ExtensionMocking: true
}
```

- Default is true.
- I would introduce this new `qmate`  options object to the config and in the near future, move all qmate related arguments to the same object for better transparency.